### ISSUE TITLE
Compress stream length component size

### ISSUE THREAD

**Issue by TomAFrench** [sablierhq/safe-app#9]
Currently we have several `Select` boxes for each time increment in the stream creation UI, this is contributing to us overflowing the available area (and doesn't look great besides).

One way to handle this is to pull in the equivalent component from [pay.sablier.finance](pay.sablier.finance)

![Screenshot_2020-05-29_19-29-27](https://user-images.githubusercontent.com/15848336/83292856-bfc10400-a1e2-11ea-934e-d0f34a0e6688.png)

@PaulRBerg are you happy for us to pull that component across and restyle it to match the gnosis style?

**Comment by TomAFrench** [comment]
I've had a chat with one of the Gnosis guys on their discord and they're going to be looking at relaxing the current hard 476px limit (I'm currently expecting to end up somewhere around 550px after this change).

**Comment by PaulRBerg** [comment]
> are you happy for us to pull that component across and restyle it to match the gnosis style?

Absolutely, I'll take care of this. It'd be cool if I managed to share that component as part of a components library on npm, but I'm not yet sure if I'll have time for htat.

> I've had a chat with one of the Gnosis guys on their discord and they're going to be looking at relaxing the current hard 476px limit

Great. Is it that now you can't even scroll apps? You are literally forced to fit all vertical content in 476px?

**Comment by TomAFrench** [comment]
If we overflow the boundaries then we'll get scrollbars on the iframe. We don't have to worry too much during development but obviously we we don't want this at ship time.
